anacron/runjob.c

```diff
@@ -104,9 +104,53 @@ username(void)
 static void
 xputenv(const char *s)
 {
-    char *copy = strdup (s);
-    if (!copy) die_e("Not enough memory to set the environment");
-    if (putenv(copy)) die_e("Can't set the environment");
+    char *name = NULL, *val = NULL;
+    char *retval;
```

Comment:
Please name this value differently - I'd expect this to be actually called eq_ptr or something like this.